config.proto

```diff
@@ -30,6 +30,9 @@ message AgentConfig {
 
     // javaagent has the configs specific to javaagent
     JavaAgent javaagent = 6;
+
+    // resources map define the static list of resources which is configured on the tracer
+    map<string, string> resources = 7;
```

Comment:
Shall we rename it to `resource_attributes`?

Reply:
ah yeah. sounds better.